src/options.ts

```diff
@@ -0,0 +1,161 @@
+import { BeautifierOptions, OptionValues } from "unibeautify";
+const commonOptions = {
+  comments: [
+    ["indent_comments"],
+    (options: OptionValues): string => {
+      if (options.indent_comments === false) {
+        return "noindent";
+      } else {
+        return "indent";
+      }
+    },
+  ],
```

Comment:
```
src/options.ts(86,7): error TS2322: Type '{ "C#": { comments: (string[] | ((options: OptionValues) => string))[]; cssinsertlines: string; e...' is not assignable to type 'BeautifierOptions'.
  Types of property '"C#"' are incompatible.
    Type '{ comments: (string[] | ((options: OptionValues) => string))[]; cssinsertlines: string; endcomma:...' is not assignable to type 'boolean | BeautifierLanguageOptionComplex | undefined'.
      Type '{ comments: (string[] | ((options: OptionValues) => string))[]; cssinsertlines: string; endcomma:...' is not assignable to type 'BeautifierLanguageOptionComplex'.
        Property 'comments' is incompatible with index signature.
          Type '(string[] | ((options: OptionValues) => string))[]' is not assignable to type 'boolean | "align_assignments" | "arrow_parens" | "brace_style" | "break_chained_methods" | "comma...'.
            Type '(string[] | ((options: OptionValues) => string))[]' is not assignable to type '[BeautifierOptionName[], BeautifierOptionTransformFunction]'.
              Property '0' is missing in type '(string[] | ((options: OptionValues) => string))[]'.
```

Reply:
Looks similar to https://github.com/Unibeautify/unibeautify/blob/master/test/beautifier/options.spec.ts#L277